### PR TITLE
Add admin container to devenv

### PR DESCRIPTION
**Pull request by grahamhayes**

### CODE REVIEW

**Comment by grahamhayes** [comment]
ah, just use #17

File: process_manifest.rb [hunk 1]
```diff
@@ -82,6 +82,11 @@ def patch_container_volumes(container)
         container_volume(name: "salt", path: SALT_ORIG_DIR),
         container_volume(name: "salt-master-config", path: "/etc/salt/master.d")
       ]
+    when "salt-master"
+      [
+        container_volume(name: "salt", path: SALT_ORIG_DIR),
+        container_volume(name: "salt-master-config", path: "/etc/salt/master.d")
+      ]
```
Comment (ereslibre) [inline process_manifest.rb]: Isn't this duplicated? we already have the case for `salt-master`.

File: manifest-templates/dev-env.yaml [hunk 2]
```diff
@@ -33,6 +52,13 @@ spec:
           mountPath: /var/run/mysql
         - name: setup-mysql
           mountPath: /setup-mysql.sh
+    - name: dev-env-admin-node
+      image: sles12/salt-minion:2016.11.4
+      volumeMounts:
+        - name: salt-admin-minion-pki
+          mountPath: /etc/salt/pki/minion
+        - name: dev-env-salt-admin-minion-config
```
Comment (ereslibre) [inline manifest-templates/dev-env.yaml]: I guess this mount name is to match with the generated host volume when using `host_volume`, right?

In this case I'd say it's better to also patch the container in the `process_manifest` script, if we change the prefix from `dev-env-` to anything else for whatever reason this mount would stop working, no?

File: process_manifest.rb (hunk 1 above)
Comment (grahamhayes) [inline process_manifest.rb]: yeap - must have been a copy pasta error :/

File: manifest-templates/dev-env.yaml (hunk 2 above)
Comment (grahamhayes) [inline manifest-templates/dev-env.yaml]: yup - fixed

**Pull request merged**